game-core/src/main/java/games/strategy/triplea/TripleAPlayer.java

```diff
@@ -536,9 +536,9 @@ public CasualtyDetails selectCasualties(final Collection<Unit> selectFrom,
   }
 
   @Override
-  public int[] selectFixedDice(final int numDice, final int hitAt, final boolean hitOnlyIfEquals, final String title,
+  public int[] selectFixedDice(final int numDice, final int hitAt, final String title,
       final int diceSides) {
```

Comment:
Would you mind de-wrapping this parameter list now that it should all fit on one line?  Ditto for the similar methods in `AbstractUi`, `DiceChooser`, and `TripleAFrame` below.

Reply:
Yup, updated